### DISCUSSION TITLE
[FEATURE REQUEST]: Onboarding Tutorial

### DISCUSSION THREAD

**Discussion by CarolinaCst** [owncloud/web#5304] [Ideas]
### Scope: Phoenix ownCloud

Actual Landing Page:
https://phoenix.owncloud.com

### Steps to reproduce

1. Open Phoenix ownCloud
2. Enter Username and Password
3. Homepage is shown

### Expected behaviour

User Stories:
- As a new user opening the Website for the first time, I expect an Onboarding Tutorial

- As a participant and new user, I prefer an Onboarding that presents an introduction and highlight the main features, actions and navigation functions of ownCloud, so that I can save time and have a basic knowledge of the site

- It would also be good to go back to the Onboarding Tutorial to check some other elements anytime I want

- The Onboarding offers the possibility to be skip

### Actual behaviour

- Participants first explore the site 

- Some users need extra time to feel comfortable and to find the features they need to use

- A few participants have completed the task in a longer period of time 

Outcome:

- Onboarding tutorial for mobile and desktop elements

**Comment by fschade** [comment]
@tbsbdr - FYI - ux topic
